bismuth.spec

```
@@ -29,19 +30,67 @@ them via keyboard, just like in classical tiling window managers
 
 %files
 %license LICENSES/MIT.txt
+%{_libdir}/qt5/qml/org/kde/bismuth/core/qmldir
+%{_libdir}/qt5/qml/org/kde/bismuth/core/libbismuth_core.so
 %{_libdir}/qt5/plugins/kcms/kcm_bismuth.so
+%{_libdir}/qt5/plugins/org.kde.kdecoration2/bismuth_kdecoration.so
 %{_datadir}/config.kcfg/bismuth_config.kcfg
-%{_datadir}/icons/hicolor/32x32/status/bismuth*
+%{_datadir}/kconf_update/bismuth_old_conf_ui.upd
+%{_datadir}/kconf_update/bismuth_shortcuts_category.upd
+%{_datadir}/kconf_update/bismuth_new_logger.upd
+%{_datadir}/kconf_update/bismuth_old_conf_ui.sh
```

We can use wildcards here
```suggestion
%{_datadir}/kconf_update/bismuth*
```